apps/storybook/src/Pan.stories.tsx

```diff
@@ -23,36 +19,53 @@ const Template: Story<TemplateProps> = (args) => {
 };
 
 export const Default = Template.bind({});
-Default.args = {
-  disabled: false,
-  modifierKey: undefined,
+
+export const ModifierKey = Template.bind({});
+ModifierKey.args = {
+  modifierKey: ['Shift'],
+};
+
+export const MultipleModifierKey = Template.bind({});
+MultipleModifierKey.args = {
+  modifierKey: ['Control', 'Shift'],
+};
+
+export const MiddleButton = Template.bind({});
+MiddleButton.args = {
+  button: [MouseButton.Middle],
+};
+
+export const TwoButtons = Template.bind({});
+TwoButtons.args = {
+  button: [MouseButton.Left, MouseButton.Middle],
 };
 
 export const Disabled = Template.bind({});
 Disabled.args = {
   disabled: true,
-  modifierKey: undefined,
-};
-
-export const ModifierKey = Template.bind({});
-ModifierKey.args = {
-  disabled: false,
-  modifierKey: 'Shift',
 };
 
 export default {
-  ...VisCanvasStoriesConfig,
   title: 'Building Blocks/PanZoom/Pan',
-  parameters: {
-    ...VisCanvasStoriesConfig.parameters,
-    controls: {
-      include: ['disabled', 'modifierKey'],
-    },
+  component: Pan,
+  parameters: { layout: 'fullscreen' },
+  decorators: [FillHeight],
+  args: {
+    button: [MouseButton.Left],
+    modifierKey: [],
+    disabled: false,
   },
   argTypes: {
+    button: {
+      control: {
+        type: 'inline-check',
+        labels: { [MouseButton.Left]: 'Left', [MouseButton.Middle]: 'Middle' },
```

Comment:
I have to explicitly set the labels of the options, as by default it uses the values of the enum (`0`, `1`)

Reply:
Good to know :+1: